src/app/layout.tsx

```diff
@@ -20,9 +21,11 @@ export default function RootLayout({
       <body>
         <Providers>
           <main>
-            <AppShell>
-              {children}
-            </AppShell>
+            <NextUIProvider locale='nb-NO'>
```

Comment:
Provides ikke NextUIProvider fra <Providers> to linjer opp?

Reply:
Pusha fix!